tests/test_invoices_summary.py

```diff
@@ -0,0 +1,32 @@
+# encoding: utf-8
```

Comment:
Why not add this test to `test_invoices.py`?

Reply:
Oh yes, makes a lot of sense. Thank you!